app/mm-redux/selectors/entities/threads.ts

```diff
@@ -52,7 +52,7 @@ export function getThread(state: GlobalState, threadId: $ID<UserThread>, fallbac
     if (!thread || !thread?.id) {
         if (fallbackFromPosts) {
             const post = getPost(state, threadId);
-            if (post?.participants?.length) {
+            if (post && !post?.root_id) {
```

Comment:
nit: the `?` is redundant in this case
```suggestion
            if (post && !post.root_id) {
```